src/renderers/shared/shared/__tests__/ReactUpdates-test.js

```diff
@@ -681,20 +682,21 @@ describe('ReactUpdates', () => {
       }
 
       componentDidMount() {
+        this.container = document.createElement('div');
```

Comment:
It's allowed to render into an empty element. This test intentionally tests that so your change messes that up.